docs/csharp/language-reference/compiler-messages/array-declaration-errors.md

```diff
@@ -110,6 +120,11 @@ That's by design. The text closely matches the text of the compiler error / warn
 - [**CS9188**](#invalid-collection-builder): *Type has a CollectionBuilderAttribute but no element type.*
 - [**CS9203**](#invalid-collection-initializer): *A collection expression of this type cannot be used in this context because it may be exposed outside of the current scope.*
 - [**CS9210**](#invalid-collection-initializer): *This version of <xref:System.Collections.Immutable.ImmutableArray%601?displayProperty=nameWithType>cannot be used with collection expressions.*
+- [**CS9212**](#invalid-collection-initializer): *Spread operator '`..`' cannot operate on variables of type 'type' because 'type' does not contain a public instance or extension definition for 'member'*
```

Comment:
```suggestion
- [**CS9212**](#invalid-collection-initializer): *Spread operator '`..`' cannot operate on variables of type 'type' because 'type' does not contain a public instance or extension definition for 'member'*.
```